web/packages/teleport/src/Notifications/Notification.tsx

```diff
@@ -96,6 +96,27 @@ export function Notification({
     hideNotificationAttempt.status === 'processing' ||
     (view === 'Unread' && clicked)
   ) {
+    // If this is a text content notification, the dialog should still be renderable. This is to prevent the text content dialog immediately disappearing
```

Comment:
This doesn't seem related to tctl..

Reply:
This is relevant because this fixes a bug causing text content notifications to be unreadable when opened from the "unread" tab, and text content notifications can only ever be generated by `tctl notifications create`. The UI part of this feature would be broken without this